apps/dashboard/config/configuration_singleton.rb

```diff
@@ -370,6 +370,14 @@ def file_download_dir_max
     ENV['OOD_DOWNLOAD_DIR_MAX']&.to_i || 10737418240
   end
 
+  # The maximum size of a file that can be edited in the file editor.
+  #
+  # Default for OOD_FILE_EDITOR_MAX is 12*1024*1024 bytes.
+  # @return [Integer]
+  def file_editor_max
+    ENV['OOD_FILE_EDITOR_MAX']&.to_i || 12582912 
```

Comment:
Could the environment variable mention "SIZE" as well? I feel like "MAX" is a bit unclear. Even if it mirrors the naming scheme of `OOD_DOWNLOAD_DIR_MAX`, I don't see that as a good reason to repeat previous mistakes.

I.e., I'd suggest `OOD_FILE_EDITOR_MAX_SIZE` as the name.

12 MiB sounds like a good default!